### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Subscriptions_Screen Reader: NVDA/Narrator is not reading about check box information when we navigate to list items present inside "Filter" combo box

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2676]
[2676_Filter_NVDA issue.zip](https://github.com/microsoft/vscode-azurefunctions/files/6019989/2676_Filter_NVDA.issue.zip)
[Check out Accessibility Insights! -](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Actual Result:
NVDA/Narrator is not reading about check box information when we navigate to list items present inside "Filter" combo box

### Expected Result:
NVDA/Narrator should read about check box information when we navigate to list items present inside "Filter" combo box. It should read the state checked/un checked

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yTCS;#BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Screen Reader; #WCAG1.3.1; #NVDA; #Win10; #Element:Listitem; #Narrator;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Turn on NVDA/Narrator
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Tab to Tree view and use down arrow to navigate to tree items.
9. Tab to "filter" and press enter on it.
10. Navigate to list items and verify whether NVDA is reading about the checkbox information or not.

### User Impact:
Screen Reader users will not know the functionality of controls if screen reader is not giving complete information

### Attachment for Reference:

![2676_Narrator issue](https://user-images.githubusercontent.com/38523250/108309036-d37c2280-71d6-11eb-84c0-e0667440624e.png)

[2676_Filter_NVDA issue.zip](https://github.com/microsoft/vscode-azurefunctions/files/6019999/2676_Filter_NVDA.issue.zip)

**Comment by ejizba** [comment]
This should be filed directly on VS Code as I can repro with a simple quick pick like this:
```
await vscode.window.showQuickPick(['hello', 'world'], { canPickMany: true, ignoreFocusOut: true });
```
